<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fereastraInregistrare</class>
 <widget class="QDialog" name="fereastraInregistrare">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#fereastraInregistrare
{
background:url(:/imaginebuna.png)
}</string>
  </property>
  <widget class="QLabel" name="labelInregistrare">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>INREGISTRARE</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>271</width>
     <height>371</height>
    </rect>
   </property>
   <widget class="QLabel" name="labelNume">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Nume:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNume">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPrenume">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelPrenume">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Prenume:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditUsername">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelUsername">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Username:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditParola">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelParola">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Parola:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCNP">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>240</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelCNP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>CNP:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMail">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>290</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelMail">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Mail:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditTelefon">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>340</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelTelefon">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Telefon:</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonInapoi">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>60</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Inapoi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonInregistrare">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>460</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Inregistrare</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelInformatie">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>430</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
